packages/pods-core/src/builtin/EmailPod.ts

```diff
@@ -15,49 +17,46 @@ export class EmailPublishPod extends PublishPod<EmailPublishConfig> {
   static id: string = ID;
   static description: string = "publish to email";
 
-  get config() {
-    return super.config.concat([
-      {
-        key: "from",
-        description: "from address",
-        type: "boolean",
-        default: false,
-        example: "you <username@outlook.com>",
+  get config(): JSONSchemaType<EmailPublishConfig> {
+    return PodUtils.createPublishConfig({
+      required: [],
+      properties: {
+        from: {
+          description: "from address",
+          type: "boolean",
```

Comment:
I know this is copy /paste from previous, but shouldn't these types on to/from address be strings and not booleans?

Reply:
well yeah, that makes sense. Will update it, although this pod might need other changes as well since it is not in action currently.